vector/build.gradle

```diff
@@ -441,6 +446,8 @@ dependencies {
     implementation 'com.vanniktech:emoji-material:0.7.0'
     implementation 'com.vanniktech:emoji-google:0.7.0'
 
+    implementation 'im.dlg:android-dialer:1.2.5'
```

Comment:
Please add an entry in the file `open_source_licenses.html` for this lib

Reply:
Weird I think I already did that 🤔

Reply:
Yes it's already in

Reply:
Ok, my bad, thanks